anime_downloader/sites/animixplay.py

```diff
@@ -25,18 +26,43 @@ def search(cls, query):
     def _scrape_episodes(self):
         url = self.url
         soup = helpers.soupify(helpers.get(url))
-        ep_list = soup.find('div', {'id':'epslistplace'}).get_text()
-        logger.debug(ep_list)
-        jdata = json.loads(ep_list)
-        keyList = list(jdata.keys())
-        del keyList[0]
-        logger.debug(keyList)
-        return [jdata[x] for x in keyList]
+        # v1 and v3 is embedded video player
+        # v2 and v4 is json post request
+        if '/v2/' in self.url or '/v4/' in self.url:
+            # Uses the id in the url and encodes it twice
+            # NaN and N4CP9Eb6laO9N are permanent encoded variables found in
+            # https://animixplay.com/assets/v4.min.js
+            url_id = str.encode(self.url.split("/")[4])
+            post_id = f'NaN{base64.b64encode(url_id).decode()}N4CP9Eb6laO9N'.encode()
+            post_id = base64.b64encode(post_id).decode()
+            data_id = 'id2' if '/v4/' in self.url else 'id'
+            data = (helpers.post('https://animixplay.com/raw/2ENCwGVubdvzrQ2eu4hBH',
+                data={data_id:post_id}).json())
+            logger.debug(data)
+            if '/v4/' in self.url:
+                return [x for x in data]
+            elif '/v2/' in self.url:
```

Comment:
Implementing v2 is a not a challenge, will do it myself when I'm done/bored with MAL implementation

Reply:
ok